attr_encrypted.gemspec

```diff
@@ -7,62 +7,47 @@ require 'attr_encrypted/version'
 require 'date'
 
 Gem::Specification.new do |s|
-  s.name    = 'attr_encrypted'
-  s.version = AttrEncrypted::Version.string
-  s.date    = Date.today
+  s.name        = 'attr_encrypted'
+  s.version     = AttrEncrypted::Version.string
 
   s.summary     = 'Encrypt and decrypt attributes'
   s.description = 'Generates attr_accessors that encrypt and decrypt attributes transparently'
 
-  s.authors   = ['Sean Huber', 'S. Brent Faulkner', 'William Monk', 'Stephen Aghaulor']
-  s.email    = ['seah@shuber.io', 'sbfaulkner@gmail.com', 'billy.monk@gmail.com', 'saghaulor@gmail.com']
-  s.homepage = 'http://github.com/attr-encrypted/attr_encrypted'
-  s.license = 'MIT'
+  s.authors     = ['Mike Vastola', 'Rob Law', 'Sean Huber', 'S. Brent Faulkner', 'William Monk', 'Stephen Aghaulor']
+  s.email       = ['open-source@dailypay.com']
+  s.license     = 'MIT'
 
-  s.has_rdoc = false
-  s.rdoc_options = ['--line-numbers', '--inline-source', '--main', 'README.rdoc']
+  s.metadata = {
+    'homepage_uri'      => "https://github.com/attr-encrypted/attr_encrypted",
+    'source_code_uri'   => "https://github.com/attr-encrypted/attr_encrypted",
+    'changelog_uri'     => "https://github.com/attr-encrypted/attr_encrypted/blob/master/CHANGELOG.md",
+    'bug_tracker_uri'   => "https://github.com/attr-encrypted/attr_encrypted/issues",
+    'documentation_uri' => "https://rubydoc.info/gems/attr_encrypted",
+    'wiki_uri'          => "https://github.com/attr-encrypted/attr_encrypted/wiki",
+  }
 
-  s.require_paths = ['lib']
+  s.rdoc_options     = %w[--line-numbers --inline-source --main README.rdoc]
+  s.extra_rdoc_files = %w[README.md]
+  s.require_paths    = ['lib']
 
-  s.files      = `git ls-files`.split("\n")
-  s.test_files = `git ls-files -- test/*`.split("\n")
+  s.files            = `git ls-files`.split("\n")
+  s.test_files       = `git ls-files -- test/*`.split("\n")
 
   s.required_ruby_version = '>= 2.0.0'
```

Comment:
2.6 should be minimum now.
2.5 if you want to push it, but there are no good reason to still support anything below.